src/test/java/org/cp/confisual/plantuml/PlantUmlVisitorTest.java

```diff
@@ -0,0 +1,68 @@
+package org.cp.confisual.plantuml;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.cp.confisual.nevisauth.AuthState;
+import org.cp.confisual.nevisauth.Domain;
+import org.cp.confisual.nevisauth.Entry;
+import org.cp.confisual.nevisauth.PlantUmlVisitor;
+import org.cp.confisual.nevisauth.Transition;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PlantUmlVisitorTest {
```

Comment:
add more tests:
 - can visit domain where a single state is used in multiple places
 - can visit domain where there is loop is auth state flow